Software/Development/SemesterProject/App/digitalInputManager.c

```diff
@@ -40,9 +42,12 @@ static struct
  */
 void initInputs(void)
```

Comment:
Is it a good idea to hardcode the specific pin numbers? 
As we talked about create a header file where you define the pins